### PR TITLE
Mismatch name experimental features

### PR DESCRIPTION
**Pull request by wsdwsd0829**

### CODE REVIEW

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Any source code changes should have an entry in CHANGELOG.md or have #trivial in their title.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Tests/ASConfigurationTests.m [hunk 1]
```diff
@@ -55,16 +55,48 @@ - (void)textureDidActivateExperimentalFeatures:(ASExperimentalFeatures)feature
...
-  ASExperimentalFeatures expected = ASExperimentalGraphicsContexts | ASExperimentalTextNode;
+  NSArray *names = @[@"exp_graphics_contexts",
+                     @"exp_text_node",
+                     @"exp_interface_state_coalesce",
+                     @"exp_unfair_lock",
+                     @"exp_infer_layer_defaults",
+                     @"exp_network_image_queue",
+                     @"exp_collection_teardown",
+                     @"exp_framesetter_cache",
+                     @"exp_skip_clear_data"];
```
Comment (nguyenhuy) [inline Tests/ASConfigurationTests.m]: Shouldn't this test include a bad name (e.g `__invalid_name`)?

Comment (wsdwsd0829) [reply Tests/ASConfigurationTests.m]: done

File: Tests/ASConfigurationTests.m [hunk 2]
```diff
@@ -55,17 +55,108 @@ - (void)textureDidActivateExperimentalFeatures:(ASExperimentalFeatures)feature
...
+                                    ASExperimentalLayerDefaults |
+                                    ASExperimentalNetworkImageQueue |
+                                    ASExperimentalCollectionTeardown |
+                                    ASExperimentalFramesetterCache |
+                                    ASExperimentalSkipClearData;
   XCTAssertEqual(expected, ASExperimentalFeaturesFromArray(names));
 }
 
+- (void)testFlagMatchName
+{
```
Comment (wsdwsd0829) [inline Tests/ASConfigurationTests.m]: to cover shift number must comes with names update cases.

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Any source code changes should have an entry in CHANGELOG.md or have #trivial in their title.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by Adlai-Holler** [comment]
Looks good, could you add a changelog entry?

**Comment by wsdwsd0829** [comment]
Done

**Comment by Adlai-Holler** [comment]
@garrettmoon @nguyenhuy Could you guys approve the build when you have a sec?

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/345a3c58af02bad2c194142ea33ba81c/raw/7ccf55d40bd984316eade5e5007db2614718f846/log.txt)

**Pull request merged**